common/utilities/timer.h

```diff
@@ -0,0 +1,112 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+// Helper classes to keep track of time
+#pragma once
+
+#include <chrono>
+namespace time_utilities
+{
+    using clock = std::chrono::steady_clock;
+
+    class stopwatch
+    {
+    public:
+        stopwatch()
+        {
+            _start = clock::now();
+        }
+
+        // Reset the stopwatch time
+        void reset(clock::time_point start_time = clock::now()) const 
+        { 
+            _start = start_time; 
+        }
+
+        // Get elapsed in milliseconds since timer creation
+        double elapsed_ms() const
+        {
+            return std::chrono::duration_cast<std::chrono::duration<double, std::milli>>(elapsed()).count();
+        }
+
+        // Get elapsed since timer creation
+        clock::duration elapsed() const
+        {
+            return clock::now() - _start;
+        }
+
+        // Get current absolute time
+        clock::time_point now() const
```

Comment:
Why is this needed?

Reply:
It's not, it was an existing function.
I will remove it.